### ISSUE TITLE
Document example for @wrapper & multiple-method interface

### ISSUE THREAD

**Issue by rahulmutt** [typelead/eta#561]

**Comment by rahulmutt** [comment]
@jneira Looks good! Interested in submitting a PR?

**Comment by jneira** [comment]
Of course! I'll try to add at the end of section in https://eta-lang.org/docs/user-guides/eta-user-guide/java-interop/java-advanced-ffi#working-with-java-interfaces
